fdbserver/ptxn/test/Driver.h

```diff
@@ -93,6 +93,10 @@ void verifyMutationsInRecord(std::vector<CommitRecord>& record,
                              const std::vector<MutationRef>& mutations,
 							 std::function<void(CommitValidationRecord&)> validateUpdater);
 
+std::shared_ptr<TestDriverContext> initTestDriverContext();
+
+void startFakeProxy(std::vector<Future<Void>>& actors, std::shared_ptr<TestDriverContext> pTestDriverContext);
```

Comment:
Do we have a reason exposing these functions? I think they are used internally by Driver.cpp at this stage.

Reply:
Yes, that's the purpose.